<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_below">
      <item row="0" column="0">
       <widget class="QTextBrowser" name="textBrowser_evt_log"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_middle"/>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_up">
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_macs"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btn_stop_disconnect">
        <property name="text">
         <string>Disconnect Dev</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>eye open close</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_eye_open_close">
           <property name="text">
            <string>Eye Open Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_open_com">
        <property name="text">
         <string>Open COM</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_close_com">
        <property name="text">
         <string>Close COM</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_con_dev">
        <property name="text">
         <string>Connect Dev</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_serialPort"/>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>motor imagine</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btn_mi_train">
           <property name="text">
            <string>Motor Imagine Train</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_generate_model">
           <property name="text">
            <string>Motor Imagine Generate Model</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_mi_test">
           <property name="text">
            <string>Motor Imagine Test</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>cursor control</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btn_curctrl_train">
           <property name="text">
            <string>Cursor Control Train</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_generate_curctrl_model">
           <property name="text">
            <string>Cursor Control Generate Model</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_curctrl_task">
           <property name="text">
            <string>Cursor Control Test</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="btn_flash_led"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="btn_flash_led">
   <property name="text">
    <string>Flash LED</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
